CMakeLists.txt

```diff
@@ -189,4 +190,11 @@ if(DEPLOY)
   add_custom_target(deploy
     COMMAND ${CMAKE_CURRENT_BINARY_DIR}/deploy_modulefiles.sh)
   add_dependencies(deploy prep2deploy)
+  if(TCLMOD)
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/lua2tcl.py
+                   ${CMAKE_CURRENT_BINARY_DIR}/lua2tcl.py COPYONLY)
+    add_custom_target(tcl
+      COMMAND ${CMAKE_CURRENT_BINARY_DIR}/lua2tcl.py ${CMAKE_INSTALL_PREFIX}/modules)
```

Comment:
Do you need to copy this script to binary dir, can't you run it from  `CMAKE_CURRENT_SOURCE_DIR` directly?

Reply:
You can run it from everywhere, as long as you provide the full path (relative or absolute) to the installed modules.

Reply:
One can run from the source dir.  I like to copy from source to binary before I make changes (if needed). 
 Ideally, one would place it in `bin`, but that would be too radical for some. This is how I would do it. Feel free to make the change.

Reply:
I leave it up to you two - no preference.

Reply:
How about now?